R/pkg/R/functions.R

```diff
@@ -824,32 +835,23 @@ setMethod("initcap",
             column(jc)
           })
 
-#' is.nan
-#'
-#' Return true if the column is NaN, alias for \link{isnan}
-#'
-#' @param x Column to compute on.
+#' @details
+#' \code{is.nan}: Alias for \link{isnan}.
```

Comment:
roxygen does this by text order, I think - doesn't it make this go first, before isnan? perhaps we swap the order of code?